coalib/results/Diff.py

```diff
@@ -381,13 +381,52 @@ def add_lines(self, line_nr_before, lines):
         self._changes[line_nr_before] = linediff
 
     def change_line(self, line_nr, original_line, replacement):
-        """
+        r"""
         Changes the given line with the given line number. The replacement will
         be there instead.
+
+        Given an empty diff object:
+
+        >>> diff = Diff(['Hey there! Gorgeous.\n',
+        ...              "It's nice that we're here.\n"])
+
+        We can change a line easily:
+
+        >>> diff.change_line(1,
+        ...                  'Hey there! Gorgeous.\n',
+        ...                  'Hey there! This is sad.\n')
+        >>> diff.modified
+        ['Hey there! This is sad.\n', "It's nice that we're here.\n"]
+
+        We can even merge changes within one line:
+
+        >>> diff.change_line(1,
+        ...                  'Hey there! Gorgeous.\n',
```

Comment:
Giving the (now) WRONG first line as an argument looks so wrong and non-intuitive :( Looks like it will lead to bugs. Can I just pass some random stuff as the original text to get across what my change is?

I get that this is being done to make the diffs mergeable but it comes across really weird.